src/compiler/gcc.rs

```diff
@@ -1330,8 +1337,8 @@ mod test {
     #[test]
     fn test_parse_arguments_too_many_inputs() {
         assert_eq!(
-            CompilerArguments::CannotCache("multiple input files", None),
-            parse_arguments_(stringvec!["-c", "foo.c", "-o", "foo.o", "bar.c"], false)
```

Comment:
I think this should be two test cases?

Reply:
Do you mean a separate test case for a single multiple-input file (2 input files) and for multiple multiple-input files (3+ input files)?

Reply:
I initially misundersyood the intent, and redacted the comment. Having both definitly would be a good thing.

Reply:
Split the test into two tests, for single and multiple extra input files.